client/my-sites/plugins/plugins-list/test/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import noop from 'lodash/noop';
+
+import useMockery from 'test/helpers/use-mockery';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+import { sites } from './sites-data';
+import i18n from 'lib/mixins/i18n';
+
+describe( 'PluginsList', () => {
+	let React, ReactInjection, TestUtils;
+	let PluginsList = {};
+	let siteListMock = {
+		getSelectedSite() {
+			return sites[0];
+		}
+	};
+	useFakeDom();
+	useMockery( mockery => {
```

Comment:
Is there a chance to avoid all of those mocks? I saw them used in other plugins tests, but it seems like a lot of mocking for a single component. What do you think?